### DISCUSSION TITLE
Repository to configurations?

### DISCUSSION THREAD

**Discussion by dominikjeske** [PowerShell/Crescendo#121] [Ideas]
I'm wonder about json reuse in community. Is there any central place I can search or contribute? For example many people will write wrapper for kubectl - should everybody create them from scratch every time?

**Comment by ethanbergstrom** [comment]
Rather than sharing un-compiled JSON, it appears that the Cresendo team has an automatically applied tag of [CrescendoBuilt](https://github.com/PowerShell/Crescendo/blob/7e2bd606423880bb361b1ebffd9db47cccb831b8/Microsoft.PowerShell.Crescendo/src/Microsoft.PowerShell.Crescendo.psm1#L897), which one could use to search [PSGallery](https://www.powershellgallery.com/packages?q=Tags%3A%22CrescendoBuilt%22) for the modules or the source repositories.

I would also mention that some people (myself included) don't like working with raw JSON, especially when writing complex output handlers in PowerShell. For example: https://github.com/PowerShell/Crescendo/discussions/28#discussioncomment-506767
